pkg/config/manager_test.go

```diff
@@ -80,28 +81,37 @@ var _ = Describe(suiteName, func() {
 		Expect(config).To(Equal(expectedResult))
 	})
 
-	It("Reacts to updates of the delegated CNI configuration", func() {
-		stopChan := make(chan struct{})
-		doneChan := make(chan struct{})
-		go func(stopChannel, doneChan chan struct{}) {
-			Expect(configManager.MonitorDelegatedPluginConfiguration(stopChannel, doneChan)).To(Succeed())
-		}(stopChan, doneChan)
-
-		newCNIConfig := "{\"cniVersion\":\"0.4.0\",\"dns\":{},\"ipam\":{},\"name\":\"mycni-name\",\"type\":\"mycni\"}"
-		Expect(ioutil.WriteFile(multusConfigFilePath, []byte(newCNIConfig), userRWPermission)).To(Succeed())
-		Eventually(<-configManager.configWatcher.Events, time.Second).Should(Equal(
-			fsnotify.Event{
-				Name: multusConfigFilePath,
-				Op:   fsnotify.Write,
-			}))
-
-		bytes, err := json.Marshal(configManager.cniConfigData)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(string(bytes)).To(Equal(newCNIConfig))
+	Context("Updates to the delegate CNI configuration", func() {
+		var (
+			doneChannel chan struct{}
+			stopChannel chan struct{}
+		)
 
-		// cleanup the watcher
-		go func() { stopChan <- struct{}{} }()
-		Eventually(<-doneChan, time.Second).Should(Equal(struct{}{}))
+		BeforeEach(func() {
+			doneChannel = make(chan struct{})
+			stopChannel = make(chan struct{})
+			go func() {
+				Expect(configManager.MonitorDelegatedPluginConfiguration(stopChannel, doneChannel)).To(Succeed())
+			}()
+		})
+
+		AfterEach(func() {
+			go func() { stopChannel <- struct{}{} }()
+			Eventually(<-doneChannel).Should(Equal(struct{}{}))
+			close(doneChannel)
+			close(stopChannel)
+		})
+
+		It("Trigger the re-generation of the Multus CNI configuration", func() {
+			newCNIConfig := "{\"cniVersion\":\"0.4.0\",\"dns\":{},\"ipam\":{},\"name\":\"yoyo-newnet\",\"type\":\"mycni\"}"
+			Expect(ioutil.WriteFile(defaultCniConfig, []byte(newCNIConfig), userRWPermission)).To(Succeed())
+
+			multusCniConfigFile := fmt.Sprintf("%s/%s", multusConfigDir, multusConfigFileName)
+			Eventually(func() (string, error) {
+				bytes, err := ioutil.ReadFile(multusCniConfigFile)
```

Comment:
variable name `bytes` is overlapped with package name [bytes](https://pkg.go.dev/bytes). Could you please rename it?

Reply:
Sure; done in this [forced push](https://github.com/k8snetworkplumbingwg/multus-cni/compare/b248141bf15e374d466a730d955fe51fd9d3899a..0da544985452687d4175bf27e5b4e38892f67ddd).